src/configuration.ts

```diff
@@ -0,0 +1,27 @@
+import { ToolboxItem } from "./toolbox"
+
+export class Configuration {
+    constructor(
+        public default_toolbox_item_order: string[] = [
+            "mode select",
```

Comment:
Make this an enum that maps to constructors.
Enums: https://masteringjs.io/tutorials/fundamentals/enum
And then a dictionary that associates these keys with the actual constructor.
This will make it easier to expand item-specific configurations (i.e. making several sliderbars that take different callbacks) in the future.
Ideally the workflow would be:

1. User passes a list of enums (if there's additional configuration for an object, they pass a tuple with the toolbox key and the specific configuration)
2. The list gets parsed and the items get constructed (with custom configuration if necessary)
3. The constructed items get put in the toolbox 

The custom configuration could just be a list of arguments passed to the toolbox constructor, doesn't need to be it's own class.